rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,590 @@
+[
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-minimize-service-account-permissions",
+        "description": "M1048 - Application Isolation and Sandboxing - Minimize permissions and access for service accounts to limit impact of exploitation.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_iam_service_account AS acct THAT ASSIGNED AS rel google_iam_binding AS binding RETURN acct.displayName, count(rel)",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "LOW"
+      },
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-patch-management",
+        "description": "M1051 - Update Software - Update software regularly by employing patch management for internal enterprise endpoints and servers.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND HostAgent WITH automaticAppUpdates='OFF' OR automaticDownloadUpdates='OFF' OR automaticOsUpdates='OFF' OR automaticSecurityUpdates='OFF' OR automaticUpdates='OFF' OR autoSecurityPatchEnabled=false and autoSystemPatchEnabled=false",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-scan-for-vulnerable-services",
+        "description": "M1016 - Vulnerability Scanning - Regularly scan the internal network for available services to identify new and potentially vulnerable services.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND (Device|Host) THAT PROTECTS << HostAgent WITH function=('av' OR 'anti-malware') AND active=true",
```

Comment:
this query is the same as another below.. this should be more looking for Devices/Hosts that are not being scanned by a VulnScanner like Qualys or Tenable - "Find (Device|Host) THAT !SCANS << qualys_vulnerability_manager" for example

https://docs.jupiterone.io/integrations/directory/qualys?integration-docs=data+model#mapped-relationships

Reply:
FIND (Device|Host) THAT !SCANS << (HostAgent|Service) WITH function=('av' OR 'anti-malware') OR _integrationClass = 'Scanner' 

I think is a better version of this query